### PR TITLE
Added "attribution reporting: a primer" post

### PR DESCRIPTION
**Pull request by maudnals**
See https://docs.google.com/document/d/1u8YMT-jz_3GobYBRJ9QeswDYCN3QJ-TuV-SpvhyHwqw/edit#

### CODE REVIEW

**Comment by maudnals** [comment]
Waiting on resolution on https://github.com/WICG/conversion-measurement-api/issues/170 to adapt the use cases section

**Review by samdutton — APPROVED** [review]

File: site/en/docs/privacy-sandbox/attribution-reporting-primer/index.md [hunk 1]
```diff
@@ -0,0 +1,596 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Attribution Reporting (Conversion Measurement): a primer'
```
Comment (mihajlija) [inline site/en/docs/privacy-sandbox/attribution-reporting-primer/index.md:3]:
```suggestion
title: 'Introduction to Attribution Reporting (Conversion Measurement)'
```

File: site/en/docs/privacy-sandbox/attribution-reporting-primer/index.md [hunk 2]
```diff
@@ -0,0 +1,596 @@
...
+
+- Early 2021: aggregate reports and view-through measurement are added to the proposal.
+- Early 2021: the API is renamed to "Attribution Reporting API".
+
+{% Aside 'caution' %}
+
+- This post focuses on advertising use cases, but the Attribution Reporting API can be
+  used for use cases that are unrelated to advertising.
+- The advertising use cases for this API focus on linking ad clicks or views to
+  conversions (Conversion Measurement).
```
Comment (mihajlija) [inline site/en/docs/privacy-sandbox/attribution-reporting-primer/index.md:26]:
```suggestion
- This post focuses on advertising use cases, but the Attribution Reporting API can
  serve use cases that are not related to advertising as well.
- The advertising use cases for this API focus on linking ad clicks or views to
  conversions (Conversion Measurement).  
```

File: site/en/docs/privacy-sandbox/attribution-reporting-primer/index.md [hunk 3]
```diff
@@ -0,0 +1,596 @@
...
+updated: 2021-07-09
+authors:
+  - maudn
+---
+
+{% Aside %}  
+This API is a proposal and will expand over time. This blog post describes its current
+state, and will be updated as the API evolves. {% endAside %}
+
+Last updates:
```
Comment (mihajlija) [inline site/en/docs/privacy-sandbox/attribution-reporting-primer/index.md:16]:
```suggestion
Updates:
```

File: site/en/docs/privacy-sandbox/attribution-reporting-primer/index.md [hunk 4]
```diff
@@ -0,0 +1,596 @@
...
+doesn't rely on third-party cookies or mechanisms that can be used to identify individual
+users across sites.
+
+This proposal is being incubated in the open. The proposal and discussions live in the
+[WICG GitHub repository](https://github.com/WICG/conversion-measurement-api).
+
+{% Aside %}  
+This API is part of the Privacy Sandbox, a series of proposals to satisfy third-party use
+cases without third-party cookies or other cross-site tracking mechanisms. See [Privacy
+Sandbox proposals](https://developers.chrome.com/docs/privacy-sandbox).
```
Comment (mihajlija) [inline site/en/docs/privacy-sandbox/attribution-reporting-primer/index.md:42]:
```suggestion
This API is a part of Privacy Sandbox, a series of proposals to satisfy third-party use
cases without third-party cookies or other cross-site tracking mechanisms. See [Privacy
Sandbox proposals](https://developers.chrome.com/docs/privacy-sandbox).  
```

File: site/en/docs/privacy-sandbox/attribution-reporting-primer/index.md [hunk 5]
```diff
@@ -0,0 +1,596 @@
...
+Sandbox proposals](https://developers.chrome.com/docs/privacy-sandbox).  
+{% endAside %}
+
+## Why is this API needed?
+
+Today, ad conversion measurement often relies on [third-party
+cookies](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#Third-party_cookies).
+Browsers are restricting access to third-party cookies because these can be used to track
+users across sites and hinder user privacy. This API addresses in a privacy-preserving way
+the measurement solved by third-party cookies.
```
Comment (mihajlija) [inline site/en/docs/privacy-sandbox/attribution-reporting-primer/index.md:51]:
```suggestion
users across sites and hinder user privacy. This API enables those measurements 
in a privacy-preserving way, without third-party cookies.
```

File: site/en/docs/privacy-sandbox/attribution-reporting-primer/index.md [hunk 6]
```diff
@@ -0,0 +1,596 @@
...
+  platforms](https://en.wikipedia.org/wiki/Demand-side_platform) (DSP) or [data management
+  platforms](https://en.wikipedia.org/wiki/Data_management_platform) (DMP) are likely to
+  be interested in using this API to support functionality that currently relies on
+  third-party cookies.
+- Advertisers and publishers relying on custom code for advertising or conversion
+  measurement may similarly be interested in using this API to replace existing
+  techniques.
+- Advertisers and publishers relying on adtech platforms for conversion measurement don't
+  need to use the API directly, but may be interested in understanding it if they're
+  working with adtech platforms that may integrate the API.
```
Comment (mihajlija) [inline site/en/docs/privacy-sandbox/attribution-reporting-primer/index.md:65]:
```suggestion
- Adtech platforms such as [demand-side
  platforms](https://en.wikipedia.org/wiki/Demand-side_platform) (DSP) or [data management
  platforms](https://en.wikipedia.org/wiki/Data_management_platform) (DMP) may use
  this API to support functionality that currently relies on third-party cookies.
- Advertisers and publishers relying on custom code for advertising or conversion
  measurement may use this API to replace existing techniques.
- Advertisers and publishers relying on adtech platforms for conversion measurement don't
  need to use the API directly, but may be interested in understanding it if they're
  working with adtech platforms that may integrate the API.
```

File: site/en/docs/privacy-sandbox/attribution-reporting-primer/index.md [hunk 7]
```diff
@@ -0,0 +1,596 @@
...
+
+## Participate
+
+{% Aside %}  
+**Your participation is needed!** A wide variety of conversion measurement and
+optimization use cases may need to be supported by this API. Ecosystem input is vital to
+ensure that solutions to support these use cases are discussed in the open. {% endAside %}
+
+To engage, join the discussion and/or try the API. Doing both is optimal, but you're
+welcome to join the discussion whether or not you've tried the API.
```
Comment (mihajlija) [inline site/en/docs/privacy-sandbox/attribution-reporting-primer/index.md:75]:
```suggestion
**Your participation is needed!** This API may need to support a wide variety of conversion 
measurement and optimization use cases. Ecosystem input is vital to
ensure that solutions to support these use cases are discussed in the open. {% endAside %}

To engage, join the discussion and try the API. Doing both is optimal, but you're
welcome to join the discussion whether or not you've tried the API.
```

File: site/en/docs/privacy-sandbox/attribution-reporting-primer/index.md [hunk 8]
```diff
@@ -0,0 +1,596 @@
...
+
+#### Experiment locally or with a demo
+
+1.  Enable the API locally in your browser by turning on the flag
+    `#enable-experimental-web-platform-features`. A Chrome flag is a toggle that tells
+    your browser to enable certain experimental functionalities. To turn on that flag,
+    paste `chrome://flags/#enable-experimental-web-platform-features` in Chrome's search
+    bar and click **Enable**.
+2.  Run the [demo](#demo) locally (or try the [live demo](#demo)).
+3.  [Fork the demo code](#demo) and customize it, or build your own demo from scratch.
```
Comment (mihajlija) [inline site/en/docs/privacy-sandbox/attribution-reporting-primer/index.md:114]:
```suggestion
1.  To enable the API locally in your browser, turn on the flag
    `#enable-experimental-web-platform-features`. A Chrome flag is a toggle that tells
    your browser to enable certain experimental functionalities. To turn on that flag,
    paste `chrome://flags/#enable-experimental-web-platform-features` in Chrome's search
    bar and click **Enable**.
2.  Run the [demo](#demo) locally (or try the [live demo](#demo)).
3.  [Fork the demo code](#demo) and customize it, or build your own demo from scratch.
```

File: site/en/docs/privacy-sandbox/attribution-reporting-primer/index.md [hunk 9]
```diff
@@ -0,0 +1,596 @@
...
+and ask.
+
+If you have general technical questions about your use case, consider opening an issue on
+the [Privacy Sandbox dev support
+repository](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support).  
+{% endAside %}
+
+## Demo
+
+A few demos are available for you to try:
```
Comment (mihajlija) [inline site/en/docs/privacy-sandbox/attribution-reporting-primer/index.md:143]:
```suggestion
A few demos are available for you to try.
```

File: site/en/docs/privacy-sandbox/attribution-reporting-primer/index.md [hunk 10]
```diff
@@ -0,0 +1,596 @@
...
+{% Aside %}
+
+This API is a work in progress and will evolve over time, depending on ecosystem feedback
+and input.
+
+All the features this API supports are proposals. **Each of these proposals is open to
+discussion and feedback**, including those that have an initial browser implementation
+ready.
+
+This API is being incubated and developed in the open. [Participate](#participate).
```
Comment (mihajlija) [inline site/en/docs/privacy-sandbox/attribution-reporting-primer/index.md:165]:
```suggestion
This API is being incubated and developed in the open. [Consider participating](#participate) in the discussion.
```

File: site/en/docs/privacy-sandbox/attribution-reporting-primer/index.md [hunk 11]
```diff
@@ -0,0 +1,596 @@
...
+
+Reports are sent by the browser after a delay⏤days or sometimes weeks after a conversion.
+
+{% endDetails %}
+
+### Aggregate reports
+
+<figure>
+{% Img src="image/O2RNUyVSLubjvENAT3e7JSdqSOx1/HAl0ppkoxoGCtttWDk2A.png", alt="ALT_TEXT_HERE", width="800", height="1140" %}
+  <figcaption>Aggregate reports are generated as follows: the browser matches detailed clicks or views ("attribution source events") with detailed conversion data ("attribution trigger data") defined by an adtech. Adtech-defined code runs in a worklet to define contributions that will be sent by the browser in order to be used to compute aggregate reports. Aggregation services are responsible for privately computing aggregate reports for adtech.</figcaption>
```
Comment (mihajlija) [inline site/en/docs/privacy-sandbox/attribution-reporting-primer/index.md:424]: I suggest pulling these descriptions out of the caption element, and into paragraph text. The longer captions (below) don't look good with centred formatting.

**Review by mihajlija — APPROVED** [review]

File: site/en/docs/privacy-sandbox/attribution-reporting-primer/index.md (hunk 1 above)
Comment (mihajlija) [reply site/en/docs/privacy-sandbox/attribution-reporting-primer/index.md:3]: Just a suggestion to use a more common word.

**Pull request merged**